grammar.js

```
@@ -27,6 +27,63 @@ const spacedValueContinuation = ($, atom) =>
     repeat1(atom),
   );
 
+/**
+ * The inner content of a double-quoted string, shared by the standalone rule
+ * and the immediate variant used for adjacent-segment concatenation.
```

ESLint

[error] Expected 1 lines after block description

(jsdoc/tag-lines)